File: app/src/main/java/com/example/georgeissac/mvp/domain/interfaces/UseCaseInterface.java
package com.example.georgeissac.mvp.domain.interfaces;

import com.example.georgeissac.mvp.database.CountryPojo;

import java.util.List;

public interface UseCaseInterface {

    void addCountries(List<CountryPojo> list);

    void getCountyList();

    void searchCountry(String string);

}

File: app/src/main/java/com/example/georgeissac/mvp/presentationLayer/UseCaseInterface.kt
package com.example.georgeissac.mvp.presentationLayer

import android.arch.lifecycle.LiveData
import com.example.georgeissac.mvp.domainLayer.addCountry.AddCountry
import com.example.georgeissac.mvp.domainLayer.getCountry.response.Country
import com.example.georgeissac.mvp.domainLayer.getCountryOnSearch.SearchCountry

interface UseCaseInterface {
    fun addCountries(list : List<Country>)
    fun getCountyList()
    fun searchCountry(string: String)
}